quick-setup.py

```
@@ -0,0 +1,369 @@
+    print_colored("📋 Checking Python version...", Colors.BLUE)
+    
+    version = sys.version_info
+    print(f"   Python {version.major}.{version.minor}.{version.micro} detected")
+    
+    # Issue #312 shows problems with Python 3.12, recommend 3.11
+    if version.major < 3 or (version.major == 3 and version.minor < 8):
+        print_colored(f"❌ Python 3.8+ is required. You have {version.major}.{version.minor}", Colors.RED)
+        print_colored("   Please install Python 3.8 or later.", Colors.YELLOW)
+        return False
+    elif version.major == 3 and version.minor >= 12:
+        print_colored(f"⚠️  Python 3.12+ detected. Known compatibility issues!", Colors.YELLOW)
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)